### DISCUSSION TITLE
Idea for BO - Introduce basic and advanced UI mode

### DISCUSSION THREAD

**Discussion by Hlavtox** [PrestaShop/PrestaShop#34429] [Ideas]
Just a quick idea, maybe it's nonsense, but try to think about it.

**What if we introduced two interface modes for backoffice?**

One default mode for beginner employees, new merchants or older people. One advanced mode for people who use Prestashop for some time, know where to click and don't want to click and scroll through crap.

The advanced mode would have help buttons hidden, it would have smaller paddings so more stuff fits on screen, it would have hidden button labels, when icon would be enough.

#### Screenshots
![default](https://user-images.githubusercontent.com/6097524/100139516-e5784e80-2e8f-11eb-9eba-d46794951c95.JPG)
![advanced](https://user-images.githubusercontent.com/6097524/100139518-e6a97b80-2e8f-11eb-8ffb-02b7213a9f1d.JPG)

ping @prestascott @LouiseBonnard @MatShir

**Comment by hibatallahAouadni** [comment]
Hello @Hlavtox 

Thanks for the suggestion, the Product Team will take a look.
Please stay tuned !

Ping @PrestaShop/product-team , what do you think?

PS: Could you please edit your issue to fit the feature template, thanks!
